src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactoryUnitTest.java

```diff
@@ -1468,6 +1490,15 @@ public void testCreateFuncotations(final String expectedGeneName,
                                 final ReferenceDataSource referenceDataSource,
                                 final String transcriptFastaFile, final String transcriptGtfFile) {
 
+        //        outList.addAll(
+        //        addReferenceDataToUnitTestData(DataProviderForTroubleVariants.provideTP53AllelesForOrderTestsHg19(),
+        //        FuncotatorReferenceTestUtils.retrieveHg19Chr17Ref(),
+        //        gencodeHg19TP53FeatureReader,
+        //        refDataSourceHg19Ch17,
+        //        FuncotatorTestConstants.TP53_REVERSE_ORDER_GENCODE_TRANSCRIPT_FASTA_FILE,
+        //        FuncotatorTestConstants.TP53_REVERSE_ORDER_GENCODE_TRANSCRIPT_FASTA_FILE ) );
```

Comment:
Why is this commented out?

Reply:
That was some code I forgot to remove.  I had a little trouble determining which fields were being filled in by the helper methods (hence the commented out code at line 195 to remind people making new tests).

I'll remove it.